client/components/account-balances/balances-tab-panel.tsx

```diff
@@ -71,16 +83,73 @@ const AccountBalancesTabPanel: React.FC = () => {
 			{ ( tab: BalanceTab ) => (
 				<Flex gap={ 0 } className="wcpay-account-balances__balances">
 					<BalanceBlock
-						type="available"
+						id={ `wcpay-account-balances-${ tab.currencyCode }-available` }
+						title={ fundLabelStrings.available }
 						amount={ tab.availableFunds }
 						currencyCode={ tab.currencyCode }
 						isLoading={ isLoading }
+						tooltip={
+							<BalanceTooltip
+								label={ `${ fundLabelStrings.available } tooltip` }
+								content={
+									tab.availableFunds < 0 ? (
+										<a
```

Comment:
Can we use `ExternalLink` from `@wordpress/components` here?

There is UI difference with design. It has the arrow icon.

![Screenshot 2023-04-03 at 00 19 31](https://user-images.githubusercontent.com/73803630/229368573-89c9abd2-1090-454c-a3fa-1c2620763bd9.png)

It will open the link not in the same page without specifying target="_blank" and it's used in several places, so maybe that's the standard.

I didn't think of it before but now I wonder what is considered 'external'. Is our own docs external? Or maybe it means every link that does not point to admin pages is external?

Reply:
The external icon indicates that it will open a new tab. I think there's a bug with the WordPress component where the icon surfaces an unwanted string of text. Will you check that before adding?

Reply:
Thanks for the heads up @elizaan36.

I've used ExternalLink and made sure the `(opens in new tab)` string does not display in ca83eac331b1d1e071a3f56d630bdffb831e1c45.

Please see the updated screenshots in the PR description.